### ISSUE TITLE
Unable to retrieve map data

### ISSUE THREAD

**Issue by lexural** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#68]
```
Logger: custom_components.xiaomi_cloud_map_extractor.camera
Source: custom_components/xiaomi_cloud_map_extractor/camera.py:209
Integration: xiaomi_cloud_map_extractor (documentation)
First occurred: 20:16:40 (2486 occurrences)
Last logged: 23:44:23

Unable to retrieve map data
```

```
Logger: custom_components.xiaomi_cloud_map_extractor.camera
Source: custom_components/xiaomi_cloud_map_extractor/camera.py:179
Integration: xiaomi_cloud_map_extractor (documentation)
First occurred: 20:16:44 (2485 occurrences)
Last logged: 23:44:23

Unable log in
```

```
Logger: homeassistant.components.camera
Source: helpers/entity_platform.py:603
Integration: Камера (documentation, issues)
First occurred: 7:16:47 (107 occurrences)
Last logged: 23:08:17

Updating xiaomi_cloud_map_extractor camera took longer than the scheduled update interval 0:00:05
```

```
Logger: homeassistant.helpers.entity
Source: custom_components/xiaomi_cloud_map_extractor/xiaomi_cloud_connector.py:42
First occurred: 22:59:07 (1 occurrences)
Last logged: 22:59:07

Update for camera.xiaomi_cloud_map_extractor fails
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 445, in _make_request
    six.raise_from(e, None)
  File "<string>", line 3, in raise_from
  File "/usr/local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 440, in _make_request
    httplib_response = conn.getresponse()
  File "/usr/local/lib/python3.8/http/client.py", line 1347, in getresponse
    response.begin()
  File "/usr/local/lib/python3.8/http/client.py", line 307, in begin
    version, status, reason = self._read_status()
  File "/usr/local/lib/python3.8/http/client.py", line 268, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "/usr/local/lib/python3.8/socket.py", line 669, in readinto
    return self._sock.recv_into(b)
  File "/usr/local/lib/python3.8/ssl.py", line 1241, in recv_into
    return self.read(nbytes, buffer)
  File "/usr/local/lib/python3.8/ssl.py", line 1099, in read
    return self._sslobj.read(len, buffer)
socket.timeout: The read operation timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/usr/local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 755, in urlopen
    retries = retries.increment(
  File "/usr/local/lib/python3.8/site-packages/urllib3/util/retry.py", line 531, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/usr/local/lib/python3.8/site-packages/urllib3/packages/six.py", line 735, in reraise
    raise value
  File "/usr/local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 699, in urlopen
    httplib_response = self._make_request(
  File "/usr/local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 447, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=read_timeout)
  File "/usr/local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 336, in _raise_timeout
    raise ReadTimeoutError(
urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='account.xiaomi.com', port=443): Read timed out. (read timeout=10)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 278, in async_update_ha_state
    await self.async_device_update()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 482, in async_device_update
    await task
  File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/xiaomi_cloud_map_extractor/camera.py", line 177, in update
    self._logged = self._connector.login()
  File "/config/custom_components/xiaomi_cloud_map_extractor/xiaomi_cloud_connector.py", line 90, in login
    return self.login_step_1() and self.login_step_2() and self.login_step_3()
  File "/config/custom_components/xiaomi_cloud_map_extractor/xiaomi_cloud_connector.py", line 42, in login_step_1
    response = self._session.get(url, headers=headers, cookies=cookies, timeout=10)
  File "/usr/local/lib/python3.8/site-packages/requests/sessions.py", line 555, in get
    return self.request('GET', url, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/requests/sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python3.8/site-packages/requests/sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/requests/adapters.py", line 529, in send
    raise ReadTimeout(e, request=request)
requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='account.xiaomi.com', port=443): Read timed out. (read timeout=10)
```

```
2021-01-11 23:51:18 ERROR (SyncWorker_3) [custom_components.xiaomi_cloud_map_extractor.camera] Unable log in
2021-01-11 23:51:18 WARNING (SyncWorker_3) [custom_components.xiaomi_cloud_map_extractor.camera] Unable to retrieve map data
2021-01-11 23:51:23 ERROR (SyncWorker_0) [custom_components.xiaomi_cloud_map_extractor.camera] Unable log in
2021-01-11 23:51:23 WARNING (SyncWorker_0) [custom_components.xiaomi_cloud_map_extractor.camera] Unable to retrieve map data
2021-01-11 23:51:28 ERROR (SyncWorker_2) [custom_components.xiaomi_cloud_map_extractor.camera] Unable log in
2021-01-11 23:51:28 WARNING (SyncWorker_2) [custom_components.xiaomi_cloud_map_extractor.camera] Unable to retrieve map data
2021-01-11 23:51:33 ERROR (SyncWorker_1) [custom_components.xiaomi_cloud_map_extractor.camera] Unable log in
2021-01-11 23:51:33 WARNING (SyncWorker_1) [custom_components.xiaomi_cloud_map_extractor.camera] Unable to retrieve map data
2021-01-11 23:51:38 ERROR (SyncWorker_7) [custom_components.xiaomi_cloud_map_extractor.camera] Unable log in
2021-01-11 23:51:38 WARNING (SyncWorker_7) [custom_components.xiaomi_cloud_map_extractor.camera] Unable to retrieve map data
2021-01-11 23:51:43 ERROR (SyncWorker_8) [custom_components.xiaomi_cloud_map_extractor.camera] Unable log in
2021-01-11 23:51:43 WARNING (SyncWorker_8) [custom_components.xiaomi_cloud_map_extractor.camera] Unable to retrieve map data
2021-01-11 23:51:48 ERROR (SyncWorker_11) [custom_components.xiaomi_cloud_map_extractor.camera] Unable log in
2021-01-11 23:51:48 WARNING (SyncWorker_11) [custom_components.xiaomi_cloud_map_extractor.camera] Unable to retrieve map data
2021-01-11 23:51:53 ERROR (SyncWorker_2) [custom_components.xiaomi_cloud_map_extractor.camera] Unable log in
2021-01-11 23:51:53 WARNING (SyncWorker_2) [custom_components.xiaomi_cloud_map_extractor.camera] Unable to retrieve map data
2021-01-11 23:51:58 ERROR (SyncWorker_4) [custom_components.xiaomi_cloud_map_extractor.camera] Unable log in
2021-01-11 23:51:58 WARNING (SyncWorker_4) [custom_components.xiaomi_cloud_map_extractor.camera] Unable to retrieve map data
2021-01-11 23:52:03 ERROR (SyncWorker_1) [custom_components.xiaomi_cloud_map_extractor.camera] Unable log in
2021-01-11 23:52:03 WARNING (SyncWorker_1) [custom_components.xiaomi_cloud_map_extractor.camera] Unable to retrieve map data
```

**Comment by lexural** [comment]
```
Logger: homeassistant.components.camera
Source: components/camera/__init__.py:555
Integration: Камера (documentation, issues)
First occurred: 0:02:50 (2 occurrences)
Last logged: 0:02:50

The websocket command 'camera_thumbnail' has been deprecated
```

**Comment by jonostanck** [comment]
`Logger: custom_components.xiaomi_cloud_map_extractor.camera
Source: custom_components/xiaomi_cloud_map_extractor/camera.py:179
Integration: xiaomi_cloud_map_extractor (documentation)
First occurred: 23:02:50 (32 occurrences)
Last logged: 23:05:25

Unable log in`

After last update. Before this I had problems with updating the camera after some time. Yes, China servers here...

**Comment by PiotrMachowski** [comment]
Fixed timeouts in v1.0.3, rest of logs are duplicates of existing issues
